examples/module/pcr/pointcloud.py

```diff
@@ -1,10 +1,18 @@
+'''
+Simple Test
+
+python pointcloud.py --algorithm Bunnies_before_ICP --dataroot data --save save --show
+
+'''
+
 import torch, os
 import matplotlib.pyplot as plt
 from matplotlib.lines import Line2D
 from torchvision.datasets.utils import download_and_extract_archive
+import argparse
```

Comment:
Follow the pyramid coding style mentioned in https://github.com/pypose/pypose/wiki/Developer-Guideline